pkg/jobstore/errors.go

```diff
@@ -6,20 +6,29 @@ import (
 	"github.com/bacalhau-project/bacalhau/pkg/models"
 )
 
+const JobStoreComponent = "JobStore"
 const (
-	JOB_STORE_COMPONENT = "JBS"
+	InvalidJobState   models.ErrorCode = "InvalidJobState"
+	MultipleJobsFound models.ErrorCode = "MultipleJobsFound"
+	InvalidJobVersion models.ErrorCode = "InvalidJobVersion"
```

Comment:
nit: might make more sense to call them `ConflictingJobState` and `ConflictingJobVersion` as these issues usually happen due to concurrent updates or race conditions. Users don't provide conditions when they update jobs and will be a bit confusing to say their request was invalid

Reply:
Acknowledged fixing it